streaming_jupyter_integrations/sql_utils.py

```diff
@@ -1,25 +1,71 @@
+from typing import Dict, Iterable, List
+
+import sqlparse
+
+DDL_KEYWORDS = {
+    'CREATE',
+    'DROP',
+    'ALTER',
+    'USE',
+    'LOAD',
+    'UNLOAD'
+}
+
+QUERY_KEYWORDS = {
+    'VALUES',
```

Comment:
Is there any example of query starting with VALUES ?

Reply:
According to the grammar from the docs ( https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/overview/#syntax ), query starts with `VALUES`, `WITH`, or `SELECT`. Of course, it's a response for a more general "what is a query?" question (as we can do `INSERT INTO table VALUES (...)` instead of `INSERT INTO table (SELECT ...)`) and simple `VALUES (1, 2, 3)` does not make sense.

... however, Flink does not complain about this. Just does not do anything with it:
![obraz](https://user-images.githubusercontent.com/33041023/184896871-499b6a14-f4ae-418c-9769-2870718a50de.png)